packages/mongo-nav/src/mongo-select/Trigger.tsx

```diff
@@ -106,6 +130,7 @@ export function ProjectTrigger({
   return (
     <button
       {...rest}
+      data-testid="project-trigger"
```

Comment:
"project-select-trigger"? Only reason I'm being nit-picky about these is because we will need to make a major version bump to update these IDs in the future.